docs/general-usage/model-types.rst

```diff
@@ -32,10 +33,26 @@ GraphQLField
 
         * ``source`` (string)
             You can pass a source string that is an attribute or method on the
-            class itself. If pointing to a method, all values from the instance
-            will be available within the `values` kwarg (`example <https://github.com/GrappleGQL/wagtail-grapple/blob/main/example/home/blocks.py#L99>`_).
+            class itself.
+
+            If used within a `StreamField`, the method will receive all values
+            from the instance via the `values` kwarg, e.g.:
```

Comment:
nitpick: not a kwarg 😂

Reply:
We are passing this as a kwarg, no?

e.g.

```
return lambda self, instance, info, **kwargs: getattr(
    klass, graphql_field.field_source
)(values=get_all_field_values(instance=instance, cls=cls))
```